### ISSUE TITLE
Poweramp notification issue

### ISSUE THREAD

**Issue by 5h1vu** [8bitPit/Niagara-Issues#96]
I Can't interact with Poweramp notification

**Comment by FireLight25** [comment]
Right fella, I got that issue too. (using beta 793 preview build, paid)

**Comment by 5h1vu** [comment]
Yep.. its through the notification that appears when a song starts playing in poweramp. It looks like a widget..

**Comment by jakcharvat** [comment]
So you get this?
![smartselect_20180907-211621_niagara launcher](https://user-images.githubusercontent.com/26692059/45221238-741a4880-b2e3-11e8-873c-72c409fc71dd.jpg)

**Comment by 5h1vu** [comment]
Yep . I get this.. widget. Which can't be interacted...
